app.js

```diff
@@ -3,18 +3,25 @@ const express = require('express');
 require('express-async-errors');
 const logger = require('morgan');
 const cors = require('cors');
+const TokenValidator = require('twilio-flex-token-validator').validator;
 
 const swagger = require('./swagger');
 const { apiV0 } = require('./routes');
 
 const app = express();
 const apiKey = process.env.API_KEY;
+const accountSid = process.env.TWILIO_ACCOUNT_SID;
+const authToken = process.env.TWILIO_AUTH_TOKEN;
 const version = '0.3.6';
 
 if (!apiKey) {
   throw new Error('Must specify API key');
 }
```

Comment:
I think we can remove this, and leave `apiKey` for testing only. What do you think?

Reply:
Yes.